cli-setup.sh

```diff
@@ -281,7 +260,76 @@ create_cli_config(){
     done
 }
 
+validate_platform() {
+  # check if running CentOS 7, Ubuntu 16.04, or Ubuntu 18.04
+  if [ -r /etc/centos-release ]; then
```

Comment:
Possible to make this much simpler using /etc/os-release as used here:
https://github.com/platform9/pf9-du/blob/atherton/bbone/bbslave/service_scripts/pf9-service-functions.sh#L12

Reply:
Punting this for now. Needs more testing across U16, U18, Centos7.6 and 7.7.

Reply:
Taking this in since more changes are being tested.